matplotlibcpp.h

```diff
@@ -154,6 +155,63 @@ namespace matplotlibcpp {
 		return res;
 	}
 
+	template< typename Numeric>
+	bool hist(const std::vector<Numeric>& y, long bins=10,std::string color="b", double alpha=1.0){
+
+		PyObject* ylist = PyList_New(y.size());
+		
+		PyObject* kwargs = PyDict_New();
+		PyDict_SetItemString(kwargs, "bins" ,PyFloat_FromDouble(bins));
```

Comment:
I think it should be PyLong_FromLong(), running this gives me 

    /usr/lib/python2.7/dist-packages/numpy/lib/function_base.py:600: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future